sklearn/tests/test_cross_validation.py

```diff
@@ -479,27 +478,29 @@ def assert_counts_are_ok(idx_counts, p):
         assert_counts_are_ok(train_counts, ex_train_p)
         assert_counts_are_ok(test_counts, ex_test_p)
 
-def test_predefinedsplit_with_kfold_split():       
-    '''Check that PredefinedSplit can reproduce a split generated by Kfold.'''     
-    folds = -1 * np.ones(10)       
-    kf_train = []      
-    kf_test = []       
-    for i, (train_ind, test_ind) in enumerate(cval.KFold(10, 5, shuffle=True)):        
-        kf_train.append(train_ind)     
-        kf_test.append(test_ind)       
-        folds[test_ind] = i        
-    ps_train = []      
-    ps_test = []       
-    ps = cval.PredefinedSplit(folds)       
-    for train_ind, test_ind in ps:     
-        ps_train.append(train_ind)     
-        ps_test.append(test_ind)       
-    assert_array_equal(ps_train, kf_train)     
+
+def test_predefinedsplit_with_kfold_split():
+    # Check that PredefinedSplit can reproduce a split generated by Kfold.
+    folds = -1 * np.ones(10)
+    kf_train = []
+    kf_test = []
+    for i, (train_ind, test_ind) in enumerate(cval.KFold(10, 5, shuffle=True)):
+        kf_train.append(train_ind)
+        kf_test.append(test_ind)
+        folds[test_ind] = i
+    ps_train = []
+    ps_test = []
+    ps = cval.PredefinedSplit(folds)
+    for train_ind, test_ind in ps:
+        ps_train.append(train_ind)
+        ps_test.append(test_ind)
+    assert_array_equal(ps_train, kf_train)
```

Comment:
This is interesting... why does git think of this as a whole lot of changes, when all I did was add a newline and change docstring --> comment... Are there any other representations for newline other than `'\n'` (perhaps this code was written in mac which till version 9 seems to have used `'\r'` for newline) ?

Not that the reason really matters, I am just curious :)